src/bentoml/_internal/monitoring/otlp.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import annotations
+
+import os
+import random
+import typing as t
+import logging
+import datetime
+import collections
+import logging.config
+from typing import TYPE_CHECKING
+
+from opentelemetry.sdk._logs import LoggerProvider
```

Comment:
Why do we have to use a private library like this? Don't opentelemetry has a public SDK for logs?

Reply:
Seems like they dont 😢